svcerr/svcerr.go

```diff
@@ -39,7 +40,7 @@ type lutherError struct {
 }
 
 // Error implements error.
-func (s lutherError) Error() string {
+func (s *lutherError) Error() string {
```

Comment:
A new static check caught this error where we're embedding a proto struct that itself has an embedded mutex